packages/management-api/src/managed-pod.ts

```diff
@@ -15,13 +15,22 @@ export type Management = {
   status: {
     managed: boolean
     running: boolean
-    error: boolean
+    // Optional error object - reset to undefined on successful connection
+    error?: {
+      message: string // Error message
+      code: number // HTTP error code of the error
```

Comment:
If we use comments, let's use the standard JSDoc to make them official doc info instead of staying in a temporal form.